Nonprofit Updates (#6086)
mattermost/docs: source/about/subscription.rst

Mattermost Nonprofit License program
--------------------------------------

> The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.

Reviewer: ```suggestion
The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering <about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```

> The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted `:doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.

Reviewer: ```suggestion
The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```

Who's eligible?
~~~~~~~~~~~~~~~~

Organizations applying for a non-profit license must meet **ALL** of the following requirements:

> - Be able to provide documentation that establishes a recognized nonprofit status.

Reviewer: ```suggestion
- Be able to provide documentation that establishes the organization as a recognized nonprofit status.
```

- Have no affiliation with government, academic, commercial, or religious entities.
- Be unable to afford a commercial Mattermost license.
- Be willing to let us use their logo for promotional purposes.

> - Be willing to pay a $250 processing fee for a three-year, Self-hosted Professional license contract for up to 1000 users.

Reviewer: ```suggestion
- Be willing to pay a $250 processing fee for a three-year, self-hosted Professional license contract for up to 1000 users.
```

> If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>` instead.

Author: ```suggestion
If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.
```

> If you represent an open source community or project that is not hosted under a recognized nonprofit but would like a commercial license for Mattermost, please email community@mattermost.com with your organization’s needs and requirements.

Reviewer: ```suggestion
If you represent an open source community or project that is not hosted under a recognized nonprofit, but would like a commercial license for Mattermost, please email community@mattermost.com with your organization’s needs and requirements.
```

How to apply?
~~~~~~~~~~~~~

> To apply for the Mattermost Nonprofit License, `please complete this form <https://mattermost.typeform.com/to/gbM6xGkS>`.

Author: ```suggestion
To apply for the Mattermost Nonprofit License, `please complete this form <https://mattermost.typeform.com/to/gbM6xGkS>`__.
```

> Please note that it may take up to 6-8 weeks for us to respond to your request.

Reviewer: ```suggestion
Please note that it may take up to eight weeks for us to respond to your request.
```
